website/docs/components/form/super-select/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,167 @@
+## Usage
+
+### When to use
+
+- When the list of options is large and search is required to help filter the list.
+- When multiple selections are made and displayed as a group.
+- When you require complex structured content to be displayed within the list (and trigger).
+
+### When not to use
+
+- To display a list of links (or buttons) in a menu, consider the [HDS Dropdown](/components/dropdown) component instead.
+- A list of options 5 or less, consider the [HDS Select](/components/form/select) form element.
+- As a means to filter an array of objects within a data set; e.g., when presented in a tabular format or list. Instead, consider other filtering patterns.
+
+
+## Variants
+
+### Single
+Selection is limited to a single list item.
+
+![Single selection example with list open](/assets/components/form/super-select/super-select-single-variant-text.png =504x*)
+
+### Multiple
+Multiple selections can be made within the list.
+
+![Multiple selection example with list open](/assets/components/form/super-select/super-select-multiple-variant.png =504x*)
+
+## Trigger default states
+
+**Empty**
```

Comment:
@majedelass Is this text meant to appear on its own line? Because in the rendered site it's appearing inline with the text following it. (Same with the other similar examples here.)

<img width="460" alt="image" src="https://github.com/hashicorp/design-system/assets/108769823/75e0deec-5925-4676-9c52-02d46aa7e64e">

Reply:
This was previously structured like this, but to reduce the side car size, this was removed and made into bold only.